package.json

```diff
@@ -3,16 +3,27 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
     "coveralls": "^3.0.2",
     "classnames": "^2.2.6",
     "dotenv-webpack": "^1.5.7",
+    "eslint-cli": "^1.1.1",
```

Comment:
Do we use `eslint-cli` anywhere? If we do, or if it is a dev -dependency, please add it to dev dependencies. @Estaer